install/updateVersion.sh

```diff
@@ -291,6 +335,9 @@ function update_istio_addons() {
   pushd $TEMP_DIR/templates/addons
   execute_sed "s|image: {MIXER_HUB}/\(.*\):{MIXER_TAG}|image: ${MIXER_HUB}/\1:${MIXER_TAG}|" grafana.yaml.tmpl
   execute_sed "s|image: {MIXER_HUB}/\(.*\):{MIXER_TAG}|image: ${MIXER_HUB}/\1:${MIXER_TAG}|" servicegraph.yaml.tmpl
+  execute_sed "s|image: {PROM_HUB}/\(.*\):{PROM_TAG}|image: ${PROM_HUB}/\1:${PROM_TAG}|" prometheus.yaml.tmpl
+ execute_sed "s|image: {ZIPKIN_HUB}/\(.*\):{ZIPKIN_TAG}|image: ${ZIPKIN_HUB}/\1:${ZIPKIN_TAG}|" zipkin.yaml.tmpl
```

Comment:
The indentation doesn't look consistent here, as if the two lines for zipkin are missing a space.

Reply:
Fixed it.